internal/rbd/rbd_util.go

```diff
@@ -828,14 +826,13 @@ func (rv *rbdVolume) flattenRbdImage(
 		return nil
 	}
 
-	ra, err := rv.conn.GetRBDAdmin()
+	log.DebugLog(ctx, "rbd: adding task to flatten image %s", rv)
```

Comment:
image: ?

Reply:
using `%q` instead, `:` is usually followed by an error